demo/package.json

```diff
@@ -0,0 +1,3 @@
+{
+	"type": "module"
+}
```

Comment:
`stack-trace` is ESM-only and I guess Vite (at least in the version we running in this repo) transpiles the config file down to the default module type, so we get a lovely `require() of ES Module ...`